src/modules/dashboard/sagas/utils/updateDomainReputation.ts

```diff
@@ -0,0 +1,37 @@
+import { AddressZero } from 'ethers/constants';
+
+import { ContextModule, TEMP_getContext } from '~context/index';
+import {
+  UserReputationQuery,
+  UserReputationQueryVariables,
+  UserReputationDocument,
+} from '~data/index';
+import { Address } from '~types/index';
+
+export function* updateDomainReputation(
+  colonyAddress: Address,
+  userAddress: Address,
+  domainId: number,
+) {
+  const apolloClient = TEMP_getContext(ContextModule.ApolloClient);
+
+  yield apolloClient.query<UserReputationQuery, UserReputationQueryVariables>({
+    query: UserReputationDocument,
+    variables: {
+      colonyAddress,
+      address: userAddress,
+      domainId,
+    },
+    fetchPolicy: 'network-only',
+  });
+
+  yield apolloClient.query<UserReputationQuery, UserReputationQueryVariables>({
+    query: UserReputationDocument,
+    variables: {
+      colonyAddress,
+      address: AddressZero,
```

Comment:
Just by curiosity, why we are doing it for AddressZero? 🤔

Reply:
This is to update the reputation for the whole colony as it will change after you smite a user.